_docs_integrate/01-connector-tutorial.md

```diff
@@ -8,47 +8,173 @@ In this tutorial we go through the basic steps necessary to establish a Relation
 
 The following steps include small pieces of the Connector's API documentation that, when executed, fire requests on a Connector created for testing purposes. So if you don't have an own Connector installed, feel free to use the samples directly. Otherwise you can use your own Connector either with a REST client (e.g. [Insomnia](https://insomnia.rest/) or [Postman](https://www.postman.com/)) or by using the RapiDoc documentation (`/docs/rapidoc`) hosted on your Connector (you have to [enable docs on your connector]({% link _docs_integrate/11-connector-configuration.md %}#corehttpapi) for the documentation route to work).
 
+The payloads for the requests the will be performed during this tutorial contain placeholders marked with `<...>`. Please replace them with values.
+
 # Prerequisites
 
 -   If you want to use your own Connector for executing the examples
     -   [Install the Connector]({% link _docs_integrate/10-connector-installation.md %})
     -   Make sure the [Sync Module is disabled]({% link _docs_integrate/11-connector-configuration.md %}#sync) (because in this tutorial we will synchronize manually via the HTTP endpoint)
     -   Get the API key configured for the Connector. Ask the person who installed the Connector for it.
--   You need the [Enmeshed App]({% link _pages/use.md %}) installed on your mobile device.
-
--   Diagramm
+-   You need the [Enmeshed App]({% link _pages/use.md %}) installed on your mobile device (App language set to `English`).
```

Comment:
Ich würde "App language set to English" nicht als Prerequisite sehen. Ich würde daher eher schreiben "in this tutorial we refer to the UI texts in English"